tests/raw-output/Earthfile

```diff
@@ -0,0 +1,12 @@
+VERSION 0.8
+FROM --pass-args ..+base
+
+IMPORT .. AS tests
+
+WORKDIR /test
+
+test-all:
+    DO tests+RUN_EARTHLY --earthfile=manifest.earth --target=+gha
+    RUN if ! grep -q '^::group::' earthly.output; then echo "::group:: not found"; exit 1; fi
```

Comment:
https://github.com/alexcb/acbtest has a readme, they are all in the base integration target; however, in this case since there's an explicit error message printed out this seems fine too.

Reply:
Switched to acbgrep without the messages and it seems a bit cleaner. Not sure what this `acb` is though  😁

Reply:
naming stuff is hard! I'm totally not opposed to having it renamed to something a bit more descriptive.